fabric/dataclient.go

```diff
@@ -495,170 +413,41 @@ func convertOne(fg *Fabric) ([]*eskip.Route, error) {
 				methods = append(methods, m.Method)
 
 				// AllowList per method and global default
+				//     example: oauthTokeninfoAllScope("uid", "foo.write")
 				var privs []interface{}
 				for _, priv := range m.Privileges {
 					privs = append(privs, priv)
 				}
 				if len(privs) == 0 {
```

Comment:
Shouldn`t `defaultScopePrivileges` be always appended to the required `privs`, not only in the case when method/path specific privileges are empty? Documentation states: `For example if you have a privilege section for an endpoint defined as per below, then the token will need the implicit uid scope as well as both of the defined events.write and service.write scopes before it will be allowed to access the endpoint.`

Reply:
done